File: Vocal.Model/Response/TalkResponse.cs
using System;
using System.Collections.Generic;

namespace Vocal.Model.Response
{
    public class TalkResponse
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public List<UserResponse> Users { get; set; } = new List<UserResponse>();
        public DateTime DateLastMessage { get; set; }
        public bool HasNewMessage { get; set; }
    }
}

File: src/models/response/talkResponse.ts
import {UserResponse} from './userResponse';
export class TalkResponse {
    Id: string;
    Name: string;
    HasNewMessage: boolean;
    DateLastMessage: Date;
    Users: Array<UserResponse>;
    Duration: number;
    Picture: string;
    IsWriting: boolean;
    TextWriting: string;
}